.github/workflows/koupleless_runtime_release_2.1.x.yml

```
@@ -139,14 +140,34 @@ jobs:
-            echo "release for module ${modules_in_right_jdk}"
-            mvn --batch-mode deploy -Prelease -pl ${modules_in_right_jdk} -am -amd -B -U
-            echo "release completed for module ${modules_in_right_jdk}"
+          # 过滤出需要发布的 adapter，过滤条件：adapter 版本与 pom.xml 中的版本一致
+          adapter_release_version=$(grep '<revision>' pom.xml | sed -e 's/.*<revision>\(.*\)<\/revision>.*/\1/' | tr -d ' ')
+          modules_in_release_version=""
+          for module in "${modules_in_right_jdk_array[@]}"; do
+            # 如果没有 adapter-mapping.yaml，则跳过（koupleless-adapter-configs 没有 adapter-mapping.yaml）
+            if [[ ! -f $module/conf/adapter-mapping.yaml ]]; then
+              continue
+            fi
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)